src/epcpm/staticmodbustoxls.py

```diff
@@ -157,6 +159,9 @@ def gen(self) -> typing.List[Fields]:
                 and parameter.uses_interface_item()
             )
             row.parameter_uses_interface_item = uses_interface_item
+            if parameter.access_level_uuid is not None:
```

Comment:
It's an interesting comment. I was following how it was previously done. I think the reason the if clause uses `is not None` is because of the way the variable is declared:
`access_level_uuid = attr.ib(default=None, type=typing.Union[str, bool])`
Note that `parameter.access_level_uuid` could be a string or a boolean. If it's a boolean, that can affect how the if statement interprets `if parameter.access_level_uuid` vs. `if parameter.access_level_uuid is not None`. While I don't think the variable is a boolean value for the situations I check it, it's probably a good safeguard.

Reply:
Ahhh...seems odd to have it be a string or a bool. But I suppose that's a conversation for another time. In that case then it does make sense to have it be not None.